robot_manager:
	ros__parameters:
		joint_stiffness: [100.0 100.0 100.0 100.0 100.0 100.0 100.0]
		
